recipe-box-frontend/src/main/frontend/src/components/IngredientList/UpdateIngredientQuantityPopUp.tsx

```diff
@@ -0,0 +1,68 @@
+import { ChangeEvent } from "react";
+import { ReactComponent as CloseButton } from "../../assets/icons/close.svg";
+import "./UpdateIngredientQuantityPopUp.css";
+interface UpdateIngredientQuantityPopUpProps {
+    setPopUpIngredientQuantity: React.Dispatch<React.SetStateAction<number>>;
+    popUpIngredientQuantity: number;
+    setAddPopUpToggled: React.Dispatch<React.SetStateAction<boolean>>;
+    ingredient: { id: number; unit: string };
+}
+
+export function UpdateIngredientQuantityPopup({
+    setPopUpIngredientQuantity,
+    popUpIngredientQuantity,
+    setAddPopUpToggled,
+    ingredient,
+}: UpdateIngredientQuantityPopUpProps) {
+    return (
+        <div className="add-ingredient-popup-overlay">
+            <div className="add-ingredient-popup">
+                <CloseButton
+                    className="close-button"
+                    onClick={() => {
+                        setAddPopUpToggled(false);
+                        setPopUpIngredientQuantity(0);
+                    }}
+                />
+                <div className="add-ingredient-popup-title">
+                    Type the quantity that you want to update for ingredient{" "}
+                    {ingredient.id} ({ingredient.unit})
```

Comment:
Here you have to use the ingredient name, the ingredient.id will be used for the actual BE request